### ISSUE TITLE
Sign in error due to redirect URI

### ISSUE THREAD

**Issue by t117tim** [jasonjoh/send-actionable-message#1]
I am trying to implement the code but getting an error on SIGN IN. 

I have followed all your instructions.
I F5/Run the VS and get the login window.
I enter details and press sign in.
I get the error described below.
As the message is reading the correct Application ID, I know it is making the connection correctly, but struggling with the reply URLs.

ERROR
I get a message: “Sorry, but we’re having trouble signing you in. AADSTS50011: The reply URL specified in the request does not match the reply URLs configured for the application: '8c8cc_blurredoutbyme'.

I can see in your instructions you have registered the APP, you have entered a Redirect URI of: “urn:ietf:wg:oauth:2.0:oob”. Is this material and is this preventing the code from running.
![SignInError](https://user-images.githubusercontent.com/46833712/74692998-dea41780-523d-11ea-8703-31497e1f6863.png)

**Comment by jasonjoh** [comment]
What version of Microsoft.Identity.Client are you using, and what reply URLs do you have configured?

**Comment by t117tim** [comment]
Hi Jason, thanks for the prompt reply. Curiously the elves have been in overnight and after finishing making shoes (https://americanliterature.com/author/the-brothers-grimm/fairy-tale/the-elves-and-the-shoemaker) have attended to my code - it is NOW working ! ! !

I think perhaps it was the "web" platform that it creates automatically. I thought I was deleting it but think I found it in in a second blade on Azure. 
At some random point, I also noticed that on my "mobile application" version definition, I got the same reply URL entered as your screenshot - urn:ietf:wg:oauth:2.0:oob. I'm not sure how that got there.
I also downloaded another git which logged me in using MS-graph and I'm not sure if that impacted things because I made the two changes at the same time.

I'll keep playing around and see how I get on, but thank you again for the help.
